<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="3" ignored="0" total="5" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="19964" started-at="2020-05-26T15:45:47 CDT" finished-at="2020-05-26T15:46:07 CDT">
    <groups>
      <group name="firstTest">
        <method signature="TestNGClassFeatures.languageTest()[pri:3, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="languageTest" class="testngFeatures.TestNGClassFeatures"/>
        <method signature="TestNGClassFeatures.dependTest()[pri:4, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="dependTest" class="testngFeatures.TestNGClassFeatures"/>
        <method signature="TestNGClassFeatures.logoTest()[pri:2, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="logoTest" class="testngFeatures.TestNGClassFeatures"/>
        <method signature="TestNGClassFeatures.titleTest()[pri:1, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="titleTest" class="testngFeatures.TestNGClassFeatures"/>
      </group> <!-- firstTest -->
    </groups>
    <test name="Default test" duration-ms="19964" started-at="2020-05-26T15:45:47 CDT" finished-at="2020-05-26T15:46:07 CDT">
      <class name="testngFeatures.TestNGClassFeatures">
        <test-method status="PASS" signature="setUP()[pri:0, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="setUP" is-config="true" duration-ms="11353" started-at="2020-05-26T15:45:47 CDT" finished-at="2020-05-26T15:45:58 CDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUP -->
        <test-method status="FAIL" signature="titleTest()[pri:1, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="titleTest" duration-ms="1832" started-at="2020-05-26T15:45:58 CDT" description="title of Page test" finished-at="2020-05-26T15:46:00 CDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[title not correct expected [Online Shopping site in India: Shop Online for Mobiles12, Books, Watches, Shoes and More - Amazon.in] but found [Online Shopping site in India: Shop Online for Mobiles, Books, Watches, Shoes and More - Amazon.in]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: title not correct expected [Online Shopping site in India: Shop Online for Mobiles12, Books, Watches, Shoes and More - Amazon.in] but found [Online Shopping site in India: Shop Online for Mobiles, Books, Watches, Shoes and More - Amazon.in]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at testngFeatures.TestNGClassFeatures.titleTest(TestNGClassFeatures.java:69)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="tearDown" is-config="true" duration-ms="2042" started-at="2020-05-26T15:46:00 CDT" finished-at="2020-05-26T15:46:02 CDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=titleTest status=FAILURE method=TestNGClassFeatures.titleTest()[pri:1, instance:testngFeatures.TestNGClassFeatures@51cdd8a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="titleTest()[pri:1, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="titleTest" duration-ms="1238" started-at="2020-05-26T15:46:02 CDT" description="title of Page test" finished-at="2020-05-26T15:46:03 CDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[title not correct expected [Online Shopping site in India: Shop Online for Mobiles12, Books, Watches, Shoes and More - Amazon.in] but found [Online Shopping site in India: Shop Online for Mobiles, Books, Watches, Shoes and More - Amazon.in]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: title not correct expected [Online Shopping site in India: Shop Online for Mobiles12, Books, Watches, Shoes and More - Amazon.in] but found [Online Shopping site in India: Shop Online for Mobiles, Books, Watches, Shoes and More - Amazon.in]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at testngFeatures.TestNGClassFeatures.titleTest(TestNGClassFeatures.java:69)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="tearDown" is-config="true" duration-ms="889" started-at="2020-05-26T15:46:03 CDT" finished-at="2020-05-26T15:46:04 CDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=titleTest status=FAILURE method=TestNGClassFeatures.titleTest()[pri:1, instance:testngFeatures.TestNGClassFeatures@51cdd8a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="logoTest()[pri:2, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="logoTest" duration-ms="79" started-at="2020-05-26T15:46:04 CDT" description="check characters" finished-at="2020-05-26T15:46:04 CDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="tearDown" is-config="true" duration-ms="882" started-at="2020-05-26T15:46:04 CDT" finished-at="2020-05-26T15:46:05 CDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=logoTest status=SUCCESS method=TestNGClassFeatures.logoTest()[pri:2, instance:testngFeatures.TestNGClassFeatures@51cdd8a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="languageTest()[pri:3, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="languageTest" duration-ms="1030" started-at="2020-05-26T15:46:05 CDT" description="check characters" finished-at="2020-05-26T15:46:06 CDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[assert not true expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: assert not true expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at testngFeatures.TestNGClassFeatures.languageTest(TestNGClassFeatures.java:88)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- languageTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="tearDown" is-config="true" duration-ms="559" started-at="2020-05-26T15:46:06 CDT" finished-at="2020-05-26T15:46:07 CDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=languageTest status=FAILURE method=TestNGClassFeatures.languageTest()[pri:3, instance:testngFeatures.TestNGClassFeatures@51cdd8a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="dependTest()[pri:4, instance:testngFeatures.TestNGClassFeatures@51cdd8a]" name="dependTest" duration-ms="0" started-at="2020-05-26T15:46:07 CDT" description="depends on methods check" depends-on-methods="testngFeatures.TestNGClassFeatures.languageTest" finished-at="2020-05-26T15:46:07 CDT">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestNGClassFeatures.dependTest()[pri:4, instance:testngFeatures.TestNGClassFeatures@51cdd8a] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestNGClassFeatures.dependTest()[pri:4, instance:testngFeatures.TestNGClassFeatures@51cdd8a] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:101)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dependTest -->
      </class> <!-- testngFeatures.TestNGClassFeatures -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
